docs/conf.py

```diff
@@ -193,7 +187,7 @@
     "github_user": github_repo_org,
     "github_repo": github_repo_name,
     "github_version": "main/docs/",
-    "current_version": version,
```

Comment:
You don't need to do this, you still have to resurrect the `version` variable as a string, since it's something that Sphinx uses internally and exposes as a `|version|` substitution.